### DISCUSSION TITLE
Add a preview button to the "New page" creation screen

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16515] [Feature Requests]
It would be ideal if Forem admin could preview their work when creating a new page from  `admin/customization/pages/new` prior to publishing the page.

**Bonus**

It would be even better if the page interface could include a Markdown editor similar to our article editor, with preview, toolbar, and rich text options.

**Comment by Raymond8457** [comment]
Have you did it practically. As I am doing it so to create a page for [cable installation](https://networkcableinstall.cablingcompany.co.za/network-cable-installations/) agency. But facing alot of issues. I also checked different videos from youtube but all in vain. So please if you have any sources of doing it properly. Then kindly share it with us.
